pkg/plugin/processor/builtin/impl/webhook/http_examples_test.go

```diff
@@ -85,15 +85,76 @@ value of the HTTP response's code in the metadata field ` + "`http_status`" + `.
 	//  }
 }
 
+//nolint:govet // we're using a more descriptive name of example
+func ExampleHTTPProcessor_DynamicURL() {
+	p := NewHTTPProcessor(conduit_log.Nop())
+
+	srv := newTestServer()
+	// Stop the server on return from the function.
+	defer srv.Close()
+
+	exampleutil.RunExample(p, exampleutil.Example{
+		Summary: `Send a request to an HTTP server with a dynamic URL`,
+		Description: `
+This example shows how to use the HTTP processor to use a record's ` + "`.Payload.After.name`" + ` field in the URL path,
+send it to a dummy HTTP server, and get a greeting with the name back.
+
+The response will be written under the record's ` + "`.Payload.After.response`.",
+		Config: map[string]string{
+			"request.url":   srv.URL + "/{{.Payload.After.name}}",
+			"response.body": ".Payload.After.response",
+		},
+		Have: opencdc.Record{
+			Payload: opencdc.Change{
+				After: opencdc.StructuredData{
+					"name": "foo",
+				},
+			},
+		},
+		Want: sdk.SingleRecord{
+			Payload: opencdc.Change{
+				After: opencdc.StructuredData{
+					"name":     "foo",
+					"response": []byte("hello, foo!"),
+				},
+			},
+		},
+	})
+
+	// Output:
+	// processor transformed record:
+	// --- before
+	// +++ after
+	// @@ -1,12 +1,13 @@
+	//  {
+	//    "position": null,
+	//    "operation": "Operation(0)",
```

Comment:
We should at least have a valid operation here, maybe also a position, otherwise it looks a bit strange.